site/themes/contour/assets/scss/_footer.scss

```diff
@@ -31,7 +31,7 @@ footer {
         p {
             margin: 0px;
         }
-        .copywrite {
+        .copyright {
```

Comment:
this is a style change -- the only consumer in this repository is also changed.
it's possible for someone to include this css file in an external consumer (as it's technically a public resource, and thus it's techncially an api change), but, they really shouldn't, ...